src/Character/Attribute.jsx

```diff
@@ -5,7 +5,8 @@ import CharacterContext from './CharacterContext';
 export default function Attribute({ attribute, value }) {
   const [editAttribute, setEditAttribute] = useState(false);
   const [val, setVal] = useState(value);
```

Comment:
I'm having a little bit of a hard time parsing what `val` and `nextVal` are supposed to represent here. Is there a different variable name you could choose to make it a little bit clearer?